packages/data-stores/src/starter-designs-queries/use-starter-designs-query.ts

```diff
@@ -53,7 +54,6 @@ export function useStarterDesignsQuery(
 		...queryOptions,
 	} );
 }
-
```

Comment:
Revert this change (keep the blank line)? 😄

Reply:
My bad, fixed